quickwit/rest-api-tests/scenarii/es_compatibility/0024-delete_indices.yaml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
it's a bummer these tests don't pass with `--engine elasticsearch`. I think we lack index creation to make them pass seamlessly, but it should be possible to create in the index conditionally with
```yaml
engines:
  - quickwit
# quickwit api here
---
engines:
  - elasticsearch
# es api here
  ```
and removing the api_root and _elastic prefix on other requests.

Reply:
I updated the tests to cover elastic search compatibility